packages/web-components/src/components/ic-radio-group/ic-radio-group.stories.mdx

```diff
@@ -165,21 +165,10 @@ import readme from "./readme.md";
               placeholder="Placeholder"
               label="What's your favourite type of coffee?"
             >
-            </ic-text-field>
-          </ic-radio-option>
-          <ic-radio-option
```

Comment:
Any reason for removing this third checkbox?

Reply:
I was testing to make sure it did with two but with the additional field to make it go vertical, I can add this back in, must've forgot I removed it.